src/app/courses/course-item/course-item.component.spec.ts

```diff
@@ -1,25 +1,115 @@
-import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { Course } from './../../shared/courses/course.interface';
+import { DebugElement, Component } from '@angular/core';
+import { ComponentFixture, TestBed, waitForAsync} from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
 
 import { CourseItemComponent } from './course-item.component';
+import { DurationPipe } from '../../duration.pipe';
+import { DatePipe } from '@angular/common';
+
+import { click } from './../../shared/helpers/spec/clickEvent';
```

Comment:
`click` - > `triggerClickEvent` ? what do you think?

Reply:
done